module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -62,16 +69,26 @@ namespace module::purpose {
     using message::skill::Look;
     using message::strategy::FallRecovery;
     using message::strategy::StandStill;
+    using message::support::GlobalConfig;
+    using message::input::State;
 
     Soccer::Soccer(std::unique_ptr<NUClear::Environment> environment) : BehaviourReactor(std::move(environment)) {
 
-        on<Configuration>("Soccer.yaml").then([this](const Configuration& config) {
+        on<Configuration, Trigger<GlobalConfig>>("Soccer.yaml")
+            .then([this](const Configuration& config, const GlobalConfig& global_config) {
             // Use configuration here from file Soccer.yaml
             this->log_level   = config["log_level"].as<NUClear::LogLevel>();
             cfg.force_playing = config["force_playing"].as<bool>();
 
             // Get the soccer position, if not valid option then default to striker
             cfg.position = Position(config["position"].as<std::string>());
+
+            // initialise robot storage
+            active_robots = std::vector<RobotInfo>();
+
+            // decide which player we are
+            PLAYER_ID = global_config.player_id;
```

Comment:
```suggestion
            player_id = global_config.player_id;
```
not really a const variable so snake_case

Reply:
Think it's this way in R-2-R Comms